src/redux/constants/_initialState.js

```diff
@@ -4,4 +4,118 @@ export const initialState = {
+        "heirloom tomato",
+      ],
+      requestedItems: ["Baked Goods", "Sourdough Starter", "Zucchini"],
+      images: ["tomato.jpg"],
+      ownerId: "591024",
+      ownerUsername: "shellstrop",
+      postalCode: "V3L0A5",
+      offerings: [],
+      location: "New Westminster, BC",
+    },
+    {
+      id: 3,
+      title: "Puzzle (1000 Pieces)",
+      datePosted: "2020-05-19T00:00:00.000Z",
+      description:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus eu est volutpat, luctus mauris id, bibendum sem. Mauris ultrices diam eu metus mattis, ut sodales tellus rhoncus. Nunc eleifend rutrum tortor, sit amet mattis mi euismod ac. Mauris diam ante, tincidunt in mauris et, ullamcorper fermentum urna.",
+      category: "hobbies",
+      condition: "like new",
+      quantity: 1,
+      tags: ["jigsaw puzzle", "completed once"],
+      requestedItems: ["Game Boy", "Puzzle (1000 piece or more)"],
+      images: ["puzzle.jpg"],
+      ownerId: "3000",
+      ownerUsername: "iamironman",
+      postalCode: "V5K1K2",
+      offerings: [],
+      location: "Vancouver, BC",
+    },
+    {
+      id: 4,
+      title: "Sourdough",
+      datePosted: "2020-06-06T00:00:00.000Z",
+      description:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus eu est volutpat, luctus mauris id, bibendum sem. Mauris ultrices diam eu metus mattis, ut sodales tellus rhoncus. Nunc eleifend rutrum tortor, sit amet mattis mi euismod ac. Mauris diam ante, tincidunt in mauris et, ullamcorper fermentum urna.",
+      category: "food drink",
+      condition: "brand new",
+      quantity: 1,
+      tags: ["homemade bread", "sourdough"],
+      requestedItems: ["Tomatoes", "Coffee", "Cookies"],
+      images: ["bread.jpg"],
+      ownerId: "50192",
+      ownerUsername: "fatthor",
+      postalCode: "V6V1A4",
+      offerings: [],
+      location: "Richmond, BC",
+    },
+    {
+      id: 5,
+      title: "Sourdough + Tomatoes",
+      datePosted: "2020-06-16T00:00:00.000Z",
+      description:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus eu est volutpat, luctus mauris id, bibendum sem. Mauris ultrices diam eu metus mattis, ut sodales tellus rhoncus. Nunc eleifend rutrum tortor, sit amet mattis mi euismod ac. Mauris diam ante, tincidunt in mauris et, ullamcorper fermentum urna.",
+      category: "food drink",
+      condition: "brand new",
+      quantity: 1,
+      tags: [
+        "homemade bread",
+        "sourdough",
+        "freshly picked",
+        "heirloom tomato",
+      ],
+      requestedItems: ["Coffee", "Cookies", "Toilet Paper", ""],
+      images: ["bread.jpg", "tomato.jpg"],
+      ownerId: "50192",
+      ownerUsername: "fatthor",
+      postalCode: "V6V1A4",
+      offerings: [],
+      location: "Richmond, BC",
+    },
+  ],
+  currentUser: {
+    id: "591024",
+    userName: "shellstrop",
+    postalCode: "V3L0A5",
+  },
+  itemDetail: {},
```

Comment:
Note as reminder: "itemDetail" is for the itemDetail page that a user clicks into. If they are not in an itemDetail's page, it will be an empty object